python/search/service.py

```diff
@@ -50,6 +51,21 @@ def search(cls, query, country='', industry=''):
             return "Error handling request!"
         return cls.format_response(response.text)
 
+    @classmethod
+    def company_match(cls, query):
+        parameters = {
+            'per_page': 5,
+        }
+        if query:
+            parameters['q'] = query
```

Comment:
What happens if you do an api request without a query? Does the api return something reasonable?